.gitignore

```diff
@@ -0,0 +1 @@
+/target/
```

Comment:
Can you try to rebase this branch on top of the new master? This will make sure you have all the latest goodies in terms of .gitignore, etc.

Reply:
Oh yeah! will do